src/utils/update/reviewBeneficialOwners.ts

```diff
@@ -0,0 +1,82 @@
+import * as config from "../../config";
+import { ApplicationData } from "../../model";
+import { BeneficialOwnerIndividual } from "../../model/beneficial.owner.individual.model";
+import { BeneficialOwnerGov } from "../../model/beneficial.owner.gov.model";
+
+const checkBoIndividualValidation = (boi: BeneficialOwnerIndividual) => {
+  if (boi?.usual_residential_address){
+    return true;
+  } else {
+    return false;
+  }
+};
+
+const checkBoGovValidation = (boGov: BeneficialOwnerGov) => {
+  if (boGov.principal_address){
+    return true;
+  } else {
+    return false;
+  }
+};
+
+export const checkAndReviewBeneficialOwner = (appData: ApplicationData) => {
```

Comment:
I know its a draft but think we need to try and think of a better way to re-write this method to maybe loop through each BO key?? I'll start having a think in my ticket too
Something like:

``` boKeys = [ BeneficialOwnerIndividualKey, BeneficialOwnerOtherKey, BeneficialOwnerGovKey ]
BeneficialOwnerGovKeys.forEach( boKey => {
  // bla bla bla
  // appdata[boKey] = do stuff
}) ```